code/src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const ColoredLine = ({ color }) => (
+    <hr
+        style={{
+            color: color,
+            backgroundColor: color,
+            height: 0.1
+        }}
+    />
+  );
+
+export const Header = () => (
+    <>
+        <h1>New albums &amp; singles</h1>
+
+        <ColoredLine 
+        color="#a0a0a0" />

```

Comment:
I'm a little confused about why you chose to do the style inline in the component, and also pass the color in the Header via the ColoredLine component instead of putting it inside the ColoredLine component itself, and I guess most of all why it isn't all in the CSS - but it's a good demonstration of what's possible in React!